src/IceRpc.Templates/Templates/IceRpc-Slice-Client/.template.config/template.json

```diff
@@ -1,57 +1,147 @@
 {
-    "$schema":"http://json.schemastore.org/template",
-    "author":"ZeroC",
-    "classifications":[
+    "$schema": "http://json.schemastore.org/template",
+    "author": "ZeroC",
+    "classifications": [
         "IceRPC",
         "Client",
         "Slice"
     ],
-    "identity":"IceRpc.Slice.Client.0.1",
-    "name":"IceRPC + Slice Client",
-    "description":"A project template for creating an IceRPC + Slice client console application.",
-    "shortName":"icerpc-slice-client",
-    "tags":{
-        "language":"C#",
-        "type":"project"
+    "identity": "IceRpc.Slice.Client.0.1",
+    "name": "IceRPC + Slice Client",
+    "description": "A project template for creating an IceRPC + Slice client console application.",
+    "shortName": "icerpc-slice-client",
+    "tags": {
+        "language": "C#",
+        "type": "project"
     },
-    "sourceName":"IceRpc-Slice-Client",
-    "preferNameDirectory":true,
-    "defaultName":"Client",
-    "symbols":{
-        "skipRestore":{
-            "type":"parameter",
-            "datatype":"bool",
-            "description":"If specified, skips the automatic restore of the project on create.",
-            "defaultValue":"false"
+    "sourceName": "IceRpc-Slice-Client",
+    "preferNameDirectory": true,
+    "defaultName": "Client",
+    "symbols": {
+        "skipRestore": {
+            "type": "parameter",
+            "datatype": "bool",
+            "description": "If specified, skips the automatic restore of the project on create.",
+            "defaultValue": "false"
+        },
+        "transport": {
```

Comment:
My preference would be to have a simpler --quic option. Not a transport choice.

Reply:
I prefer the verbosity of `--transport quic` it makes it crystal clear exactly what the options are quic or tcp with it defaulting to tcp. Im okay with changing it to `--quic` if everyone thinks that is cleaner though.